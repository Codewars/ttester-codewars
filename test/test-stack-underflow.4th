\ Copyright 2019 nomennescio
s" stack underflow" describe#{
  s" single test" it#{
    <{ drop -> 0 }>
  }#
  s" double test" it#{
    <{ drop -> 0 }>
    <{ drop drop -> 0 }>
  }#
  s" double test" it#{
    <{ drop -> 0 0 }>
    <{ drop drop -> 0 0 }>
  }#
  s" double test" it#{
    <{ drop -> 0 0 }>
    <{ drop drop -> 0 }>
  }#
}#
