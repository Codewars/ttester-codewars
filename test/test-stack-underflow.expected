
<DESCRIBE::>stack underflow

<IT::>single test

<FAILED::>Wrong number of results, expected 1 , got a 1 cell stack underflow

<COMPLETEDIN::>0.033 ms

<IT::>double test

<FAILED::>Wrong number of results, expected 1 , got a 1 cell stack underflow

<FAILED::>Wrong number of results, expected 1 , got a 2 cell stack underflow

<COMPLETEDIN::>1565296144893.672 ms

<IT::>double test

<FAILED::>Wrong number of results, expected 2 , got a 1 cell stack underflow

<FAILED::>Wrong number of results, expected 2 , got a 2 cell stack underflow

<COMPLETEDIN::>1565296144893.759 ms

<IT::>double test

<FAILED::>Wrong number of results, expected 2 , got a 1 cell stack underflow

<FAILED::>Wrong number of results, expected 1 , got a 2 cell stack underflow

<COMPLETEDIN::>1565296144893.836 ms

<COMPLETEDIN::>0.311 ms
